### PR TITLE
feat(Illustrations): Add computer lock illustration

### PR DESCRIPTION
**Pull request by ahmed-abdallah**
### Overview

Adds new computer + lock illustration

### PR Checklist

- [x] Related to designs: [Figma](https://www.figma.com/file/oQb1a4kuzvlI925Oi0LgPj/Free-trial-onboarding---Guided-exploration?node-id=1593%3A24844&mode=dev)
- [x] Related to JIRA ticket: [FLOW-514]
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [ ] This PR includes testing instructions tests for the code change
- [ ] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

1. Go to [preview](https://64934ad12547ba06ef15a5f6--gamut-preview.netlify.app/?path=/docs/atoms-illustrations--editable-illustration)
2. Check the illustration!

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](http://www.google.fr/ 'Named link title')  | [Portal Env](http://www.google.fr/ 'Named link title')   |
| Another Repo | [Another Link](http://www.google.fr/ 'Named link title') | [Another Env](http://www.google.fr/ 'Named link title')  |

[FLOW-514]: https://codecademy.atlassian.net/browse/FLOW-514?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.38.1-alpha.15ef5a.0<br/>@codecademy/gamut-kit@0.6.330-alpha.15ef5a.0<br/>@codecademy/gamut-labs@35.15.3-alpha.15ef5a.0<br/>@codecademy/gamut@55.1.8-alpha.15ef5a.0<br/>@codecademy/monaco@0.7.34-alpha.15ef5a.0<br/>@codecademy/styleguide@66.1.12-alpha.15ef5a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://649347817f9538008807267f--gamut-preview.netlify.app>https://649347817f9538008807267f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/649347817f9538008807267f>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.38.1-alpha.aebdc8.0<br/>@codecademy/gamut-kit@0.6.330-alpha.aebdc8.0<br/>@codecademy/gamut-labs@35.15.3-alpha.aebdc8.0<br/>@codecademy/gamut@55.1.8-alpha.aebdc8.0<br/>@codecademy/monaco@0.7.34-alpha.aebdc8.0<br/>@codecademy/styleguide@66.1.12-alpha.aebdc8.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://64934ad12547ba06ef15a5f6--gamut-preview.netlify.app>https://64934ad12547ba06ef15a5f6--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/64934ad12547ba06ef15a5f6>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.38.1-alpha.91807c.0<br/>@codecademy/gamut-kit@0.6.330-alpha.91807c.0<br/>@codecademy/gamut-labs@35.15.3-alpha.91807c.0<br/>@codecademy/gamut@55.1.8-alpha.91807c.0<br/>@codecademy/monaco@0.7.34-alpha.91807c.0<br/>@codecademy/styleguide@66.1.12-alpha.91807c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://649351d5edd5790c34b052b8--gamut-preview.netlify.app>https://649351d5edd5790c34b052b8--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/649351d5edd5790c34b052b8>Deploy Logs</a></p>

File: packages/gamut-illustrations/src/ComputerLock.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
...QECBAgQIECAAAECBAgQINAoIMDRCOUwAgQIECBAgAABAgQIECBAYLkCAhzLnRsjI0CAAAECBAgQIECAAAECBBoFBDgaoRxGgAABAgQIECBAgAABAgQILFdAgGO5c2NkBAgQIECAAAECBAgQIECAQKOAAEcjlMMIECBAgAABAgQIECBAgACB5QoIcCx3boyMAAECBAgQIECAAAECBAgQaBQQ4GiEchgBAgQIECBAgAABAgQIECCwXAEBjuXOjZERIECAAAECBAgQIECAAAECjQICHI1QDiNAgAABAgQIECBAgAABAgSWKyDAsdy5MTICBAgQIECAAAECBAgQIECgUUCAoxHKYQQIECBAgAABAgQIECBAgMByBQQ4ljs3RkaAAAECBAgQIECAAAECBAg0CghwNEI5jAABAgQIECBAgAABAgQIEFiugADHcufGyAgQIECAAAECBAgQIECAAIFGAQGORiiHESBAgAABAgQIECBAgAABAssVEOBY7twYGQECBAgQIECAAAECBAgQINAoIMDRCOUwAgQIECBAgAABAgQIECBAYLkCAhzLnRsjI0CAAAECBAgQIECAAAECBBoFBDgaoRxGgAABAgQIECBAgAABAgQILFdAgGO5c2NkBAgQIECAAAECBAgQIECAQKOAAEcjlMMIECBAgAABAgQIECBAgACB5QoIcCx3boyMAAECBAgQIECAAAECBAgQaBQQ4GiEchgBAgQIECBAgAABAgQIECCwXAEBjuXOjZERIECAAAECBAgQIECAAAECjQICHI1QDiNAgAABAgQIECBAgAABAgSWKyDAsdy5MTICBAgQIECAAAECBAgQIECgUUCAoxHKYQQIECBAgAABAgQIECBAgMByBQQ4ljs3RkaAAAECBAgQIECAAAECBAg0CghwNEI5jAABAgQIECBAgAABAgQIEFiugADHcufGyAgQIECAAAECBAgQIECAAIFGAQGORiiHESBAgAABAgQIECBAgAABAssVEOBY7twYGQECBAgQIECAAAECBAgQINAoIMDRCOUwAgQIECBAgAABAgQIECBAYLkCAhzLnRsjI0CAAAECBAgQIECAAAECBBoFBDgaoRxGgAABAgQIECBAgAABAgQILFdAgGO5c2NkBAgQIECAAAECBAgQIECAQKPA/weCVEAIao1SRgAAAABJRU5ErkJggg=="
```
Comment (jakemhiller) [inline packages/gamut-illustrations/src/ComputerLock.tsx:32]: is it expected that this is an image, not SVG paths?

**Review by jakemhiller — CHANGES_REQUESTED** [review]
curious about the illustrtation SVG, since it's mostly image data, is that expected?

**Comment by ahmed-abdallah** [comment]
Turns out this is a gif not an SVG at all

**Pull request closed without merging**